DependencyInjection/Compiler/ThemeCompilerPass.php

```diff
@@ -23,6 +23,8 @@ public function process(ContainerBuilder $container)
 
         $container->setAlias('templating.cache_warmer.template_paths', 'liip_theme.templating.cache_warmer.template_paths');
 
+        $container->setAlias('twig.loader', 'liip_theme.twig.loader.filesystem');
```

Comment:
this is wrong. You should not replace all loaders by the liip_theme one, because it breaks things for people using more than the Twig filesystem loader